ubuntu_image/tests/test_parser.py

```diff
@@ -1059,6 +1059,31 @@ def test_multiple_volumes_with_bootloader(self):
              for key in gadget_spec.volumes}
             )
 
+    def test_defaults_proper(self):
+        parse("""\
+defaults:
+  mfq0tsAY1HXMbx24wo0QIoFCGeLvERmX:
+    some-key: some-value
+    other-key: 42
+volumes:
+  first-image:
+    schema: gpt
+    structure:
+        - type: 00000000-0000-0000-0000-0000deadbeef
+          size: 100
+  second-image:
+    schema: gpt
+    structure:
+        - type: 00000000-0000-0000-0000-0000feedface
+          size: 200
+  third-image:
+    schema: gpt
+    bootloader: u-boot
+    structure:
+        - type: 00000000-0000-0000-0000-0000deafbead
+          size: 300
+""")
+
```

Comment:
Can you add a bit to the end of this test?  Just dig out the `defaults:snap-id:some-key` and `other-key` values and make sure they're readable.  We may not need them now, but they may have some semantics later.